app/models/events/new_discussion.rb

```diff
@@ -1,7 +1,8 @@
 class Events::NewDiscussion < Event
   def self.publish!(discussion)
     event = create!(kind: 'new_discussion',
-                    eventable: discussion)
+                    eventable: discussion,
+                    sequence_id: 0)
```

Comment:
OH! Thank you. Now I understand. Can we please change the lambda to check for discussion_id.nil? and set all the new_discussion event sequences to 0?